megamek/src/megamek/common/options/Quirks.java

```diff
@@ -160,530 +150,140 @@ public synchronized void initialize() {
-                    || qName.equals(OptionsConstants.QUIRK_POS_TRAILER_HITCH)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_ATMO_INSTABILITY)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_CRAMPED_COCKPIT)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_DIFFICULT_EJECT)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_EXP_ACTUATOR)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_FRAGILE_FUEL)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_NO_ARMS)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_NO_EJECT)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_NO_TWIST)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_LARGE_DROPPER)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_LOW_ARMS)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_POOR_LIFE_SUPPORT)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_UNBALANCED)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_WEAK_LEGS)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_WEAK_UNDERCARRIAGE)
-                    || qName.equals(OptionsConstants.QUIRK_POS_VTOL_ROTOR_COAXIAL)
-                    || qName.equals(OptionsConstants.QUIRK_POS_VTOL_ROTOR_DUAL)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_FLAWED_COOLING)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_UNSTREAMLINED)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_WEAK_HEAD_1)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_WEAK_HEAD_2)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_WEAK_HEAD_3)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_WEAK_HEAD_4)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_WEAK_HEAD_5)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_GAS_HOG)
-                    || qName.equals(OptionsConstants.QUIRK_POS_RUMBLE_SEAT)
-                    || qName.equals(OptionsConstants.QUIRK_NEG_POOR_PERFORMANCE)
-                    || qName.equals(OptionsConstants.QUIRK_POS_POWER_REVERSE)
-                    || qName.equals(OptionsConstants.QUIRK_POS_COMPACT)
-                    || qName.equals(OptionsConstants.QUIRK_POS_OVERHEAD_ARMS)   
-                    || qName.equals(OptionsConstants.QUIRK_POS_PRO_ACTUATOR)
-                    || qName.equals(OptionsConstants.QUIRK_POS_BATTLE_FIST_LA)
-                    || qName.equals(OptionsConstants.QUIRK_POS_BATTLE_FIST_RA)
-                    || qName.equals(OptionsConstants.QUIRK_POS_VESTIGIAL_HANDS_LA)
-                    || qName.equals(OptionsConstants.QUIRK_POS_VESTIGIAL_HANDS_RA)) {
-                return false;
-            }
-            return true;
+            return quirk.isAnyOf(QUIRK_NEG_HARD_PILOT, QUIRK_NEG_BAD_REP_IS, QUIRK_NEG_BAD_REP_CLAN,
```

Comment:
Not really sure this is any better for readability (from my perspective it's a little easier to have one quirk per line), but not really a deal breaker either way.

Reply:
I agree but I also strongly dislike scrolling through hundreds of lines. This is better, isn't it?

![image](https://github.com/MegaMek/megamek/assets/17069663/57ded381-7d1c-47f0-b1de-3979eef11b65)

Reply:
If the data guy can follow it, all is good :)

Reply:
Fine by me.